misk/src/test/kotlin/misk/web/actions/NotFoundActionTest.kt

```diff
@@ -180,7 +180,6 @@ class NotFoundActionTest {
   class TestModule : KAbstractModule() {
     override fun configure() {
       install(WebTestingModule())
-      install(WebActionModule.create<NotFoundAction>())
```

Comment:
no need to explicitly bind these, they are automatically bound for a Service in `MiskWebModule`. now these are resulting in dupe errors :)